src/components/QnA/comps/QueButtons.jsx

```diff
@@ -18,7 +22,7 @@ function QueButtons({ helpfulness }) {
           <HelpfulButton>Yes</HelpfulButton>
           {` (${helpfulness})`}
         </HelpfulSentence>
-        <AddAnswerButton>Add Answer</AddAnswerButton>
+        <AddAnswerButton onClick={handleClick}>Add Answer</AddAnswerButton>
```

Comment:
I don't think a `handleClick` function is needed here. We can probably just say `onClick={toggle}`.